tracer/test/Datadog.Trace.Security.IntegrationTests/Datadog.Trace.Security.IntegrationTests.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <None Remove="applicationHost.config" />
     <None Remove="xunit.runner.json" />
+    <Compile Include="..\Datadog.Trace.ClrProfiler.IntegrationTests\Debugger\LogEntryWatcher.cs" Link="LogEntryWatcher.cs" />
```

Comment:
if we are going to use it, should it be part of the `TestHelpers` maybe?

Reply:
+1

Reply:
fixed in a1f8b9359

Reply:
Last time I wanted to share code between tests there was some complex reason why it was only possible by including the source, but it doesn't seem applicable this time, as it was easy to move.